src/Kovan-DssSpell.sol

```diff
@@ -13,73 +13,208 @@
+        }
+
+        // ------------------  ESM  ------------------
+
+        // Authorize new ESM to execute in new END
+        DssExecLib.authorize(MCD_END, MCD_ESM_BUG);
+        DssExecLib.authorize(MCD_END, MCD_ESM_ATTACK);
+
+        // Authorize new ESM to execute in new VAT
+        DssExecLib.authorize(MCD_VAT, MCD_ESM_ATTACK);
+
+        // ------------------  DOG  ------------------
+
+        // Set VOW in the DOG
+        DssExecLib.setContract(MCD_DOG, "vow", MCD_VOW);
+
+        // Authorize DOG can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_DOG);
+
+        // Authorize DOG can access to VOW
+        DssExecLib.authorize(MCD_VOW, MCD_DOG);
+
+        Fileable(MCD_DOG).file("Hole", 10 * THOUSAND * RAD);
+
+
+        // --------------  CLIPPER_MOM  --------------
+
+        ClipperMomAbstract(CLIPPER_MOM).setAuthority(DssExecLib.getChangelogAddress("MCD_ADM"));
+
+        // TODO: Tolerance currently set to 50%. Verify governance parameter.
+        //   n.b. 600000000000000000000000000 == 40% tolerance
+        ClipperMomAbstract(CLIPPER_MOM).setPriceTolerance(MCD_CLIP_LINK_A, 50 * RAY / 100);
+
+
+        // ----------------  LINK-A  -----------------
+
+        // Set CLIP for LINK-A in the DOG
+        DssExecLib.setContract(MCD_DOG, "LINK-A", "clip", MCD_CLIP_LINK_A);
+
+        // Set VOW in the LINK-A CLIP
+        DssExecLib.setContract(MCD_CLIP_LINK_A, "vow", MCD_VOW);
+
+        // Set CALC in the LINK-A CLIP
+        DssExecLib.setContract(MCD_CLIP_LINK_A, "calc", MCD_CLIP_CALC_LINK_A);
+
+        // Authorize CLIP can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_CLIP_LINK_A);
+
+        // Authorize CLIP can access to DOG
+        DssExecLib.authorize(MCD_DOG, MCD_CLIP_LINK_A);
+
+        // Authorize DOG can kick auctions on CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_DOG);
+
+        // Authorize the new END to access the LINK CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_END);
+
+        // Authorize CLIPPERMOM can set the stopped flag in CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, CLIPPER_MOM);
+
+        // Authorize new ESM to execute in LINK-A Clipper
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_ESM_ATTACK);
+
+        // Whitelist CLIP in the LINK osm
+        DssExecLib.addReaderToOSMWhitelist(PIP_LINK, MCD_CLIP_LINK_A);
+
+        // Whitelist CLIPPER_MOM in the LINK osm
+        DssExecLib.addReaderToOSMWhitelist(PIP_LINK, CLIPPER_MOM);
+
+        // We can't deauthorize the FLIP yet as there might be running auctions:
+        // DssExecLib.deauthorize(MCD_CAT, MCD_FLIP_LINK_A); TODO in a future spell
+
+        // No more auctions kicked via the CAT:
+        DssExecLib.deauthorize(MCD_FLIP_LINK_A, MCD_CAT);
+
+        // No more circuit breaker for the FLIP in LINK-A:
+        DssExecLib.deauthorize(MCD_FLIP_LINK_A, DssExecLib.flipperMom());
+
+        Fileable(MCD_DOG).file("LINK-A", "hole", 10 * THOUSAND * RAD);
```

Comment:
Change this to `5 THOUSAND` in the test too.  The reason is that we want to be able to test `ilk.hole` values lower the `dog.Hole`.

```suggestion
        Fileable(MCD_DOG).file("LINK-A", "hole", 5 * THOUSAND * RAD);
```

Reply:
Done